src/components/cart/cartItem.tsx

```
@@ -0,0 +1,103 @@
+        </div>
+        <div className=''>
+          <h1>Sutotal</h1>
+        </div>
+  
+
+      </Card>
+
+
+      <div>
+        {Items.length > 0 ? (Items.map((item) => (
+              <Card className="flex items-center justify-between px-8 py-4 mb-4">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)